modules/serverless-jaeger-config.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serverless-tracing.adoc
+// */jaeger/jaeger_config/serverless-jaeger-integration.adoc
+
+[id="serverless-jaeger-config_{context}"]
+= Configuring Jaeger for use with {ServerlessProductName}
+
+.Prerequisites
+
+To configure Jaeger for use with {ServerlessProductName}, you will need:
+
+* Cluster administrator permissions on an {product-title} cluster.
+* A current installation of {ServerlessOperatorName} and Knative Serving.
+* A current installation of the Jaeger Operator.
+
+.Procedure
+
+. Create and apply the Jaeger custom resource:
++
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+  namespace: default
+EOF
+----
+
+. Enable tracing for Knative Serving, by editing the `KnativeServing` resource and adding a YAML configuration for tracing.
++
+.Tracing YAML example
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    tracing:
+      sample-rate: "0.1" <1>
+      backend: zipkin <2>
+      zipkin-endpoint: http://jaeger-collector.default.svc.cluster.local:9411/api/v2/spans <3>
+      debug: "false" <4>
+----
++
+<1> The `sample-rate` defines sampling probability. Using `sample-rate: "0.1"`` means that 1 in 10 traces will be sampled.
+<2> The backend must be `zipkin`.
+<3> The `zipkin-endpoint` must point to your `jaeger-collector` service endpoint. To get this endpoint, substitute the namespace where the Jaeger custom resource is applied.
+<4> Enabling debug mode by setting `debug: "false"` allows all spans to be sent to the server, bypassing sampling.
```

Comment:
Setting `debug: true` would enable the debug mode.  `debug: "false"` doesn't really do anything (expect pointing to users that they can enable it... )

Reply:
Oops, good catch, thanks @maschmid :slightly_smiling_face: